beluga_tutorial/scripts/visualize.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Ekumen, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Visualization Scripts for Beluga Tutorial.
+
+This script uses the matplotlib to visualize the data generated
+by beluga_tutorial example code.
+"""
+
+import yaml
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+import numpy as np
+import argparse
```

Comment:
Order alphabetically. Also, should any of these modules be included in the package.xml?

Reply:
Do you use some specific formatter when coding in python?
Every time I run Docker and the script, it doesn't ask me to install some of those libraries, but I'll try again

Reply:
Docker has its own set of dependencies installed, and some other packages do install these modules.
The formatter is installer as a pre-commit but it doesn't enforce this.

Whenever you make a list of things on multiple lines, it's good to sort them alphabetically because it makes it much easier to inspect the list to see if there's a specific item there and detect duplicate entries as they grow.